pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -512,7 +512,7 @@ func (r *RepoSyncReconciler) SetupWithManager(mgr controllerruntime.Manager, wat
 		Watches(&source.Kind{Type: withNamespace(&corev1.ServiceAccount{}, configsync.ControllerNamespace)},
 			handler.EnqueueRequestsFromMapFunc(r.mapObjectToRepoSync),
 			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{})).
-		Watches(&source.Kind{Type: withNamespace(&rbacv1.RoleBinding{}, configsync.ControllerNamespace)},
+		Watches(&source.Kind{Type: &rbacv1.RoleBinding{}},
```

Comment:
Maybe add a comment here to explain:
 `Watch RoleBindings in all namespaces, but only trigger Reconcile for RoleBindings that map to an existing RepoSync.`

Reply:
done